minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java

```diff
@@ -77,7 +77,7 @@ private LocalTachyonMaster(final String tachyonHome)
     // "mTachyonHome/tachyon*". Otherwise, it starts some distributed file system cluster e.g.,
```

Comment:
I think this block of comment is obsolete, isn't it? e.g., the "mTachyonHome/tachyon*" things seem no more true for me. Could you update it?